ModKit/UI/UI+Builders.cs

```diff
@@ -152,5 +150,23 @@ public static partial class UI {
             actions[selected].action();
             GL.EndVertical();
         }
+
+        public static void TabBar(ref int selected, bool shouldLocalize, Action header = null, params NamedAction[] actions) {
```

Comment:
This should not be necessary because the creators of NamedAction should localize the name